packages/apps/project/src/components/atoms/DatesDisplay/DatesDisplay.tsx

```diff
@@ -28,16 +28,24 @@ const DatesDisplay = ({
   fromDate?: string;
   toDate?: string;
 }) => {
+  const I18n = useTranslator();
+
   return (
     <View style={styles.dateContainer}>
-      <DateDisplay date={fromDate} />
+      <View style={styles.dateWrapper}>
+        <Text>{I18n.t('Project_StartedOn')}</Text>
+        <DateDisplay date={fromDate} />
+      </View>
```

Comment:
Create render callback as the structure is the same for both. 
Add a type in args to choose between from and to dates or ask for title and value.

If value is null, it should not render anything
Also center vertically date and title